test/measures/measures-data-spec.js

```diff
@@ -31,5 +31,40 @@ describe('measures data json', function() {
 
       assert.equal(intersection.size, multiPerformanceIds.size);
     });
+
+    describe('CAHPS measures', function() {
+      const cahpsMeasures = measuresData.filter(function(measure) {
+        var re = /CAHPS_\d+/i;
+        return measure.measureId.match(re) !== null;
+      });
```

Comment:
arrow functions are discouraged when passing as args to mocha's `describe`/`it`/`beforeEach`/`afterEach` since `this` is lexically bound (not to mocha's test runner context, which has useful helpers like `this.timeout` to configure the timeout, etc). arrow function are great for writing code tersely in a functional style though:

```js
doSomething()
  .then(foo => foo.bar)
  .then(bar => bar.baz);
```